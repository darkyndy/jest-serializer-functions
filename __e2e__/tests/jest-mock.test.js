// tested
import fixtures from './../__fixtures__/index';

// dependencies
jest.mock('./../__fixtures__/service');

describe('fixtures - jest mock', () => {
  it('function used as prop', () => {
    expect(fixtures.awesomeObj).toMatchSnapshot();
  });
});

