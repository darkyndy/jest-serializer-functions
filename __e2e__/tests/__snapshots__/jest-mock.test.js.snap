// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`fixtures - jest mock function used as prop 1`] = `
Object {
  "anonymous": [Function],
  "arrow": [Function],
  "named": [Function],
}
`;
